.github/workflows/aws_demo_runner.yml

```diff
@@ -0,0 +1,161 @@
+name: Localhost Server Demo Runner
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  build-and-run-localhost-demos:
+    name: Build and Run Localhost Server Demos
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Install Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install libmosquitto
+        run: sudo apt-get install -y libmosquitto-dev
+
+      - name: Install CMake
+        run: |
+          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
+          tar -xf cmake.tar.gz
+
+      - name: Generate SSL credentials
+        id: generate-credentials
+        uses: jasonpcarroll/CI-CD-GitHub-Actions/ssl-credential-creator@main
```

Comment:
There are a couple places where you have this pointing to your local fork. Can you update these to point to the FreeRTOS repo?